### ISSUE TITLE
Upgraded project from CMS 11.20 with EPiServer.Labs.LinkItem to 12.9 

### ISSUE THREAD

**Issue by Deepa-Puranik** [episerver/EPiServer.Labs.LinkitemProperty#10]
Hello,

We are using EPiServer.Labs.LinkItem package for single link property in our CMS 11.20 project and now we upgraded project to 12.9. According to this readme section of this page, package is deprecated in EPiServer CMS 12.6 and part of CMS. We did changes as per that link and ran script mentioned. That doesn’t make that field link item in CMS.
![image](https://user-images.githubusercontent.com/75352823/192466238-8e83969d-946e-45dd-9cec-dd61ed44b165.png)

**Comment by jsalwen** [comment]
Did you run the DB update script? We are moving from 12.4 to 12.11 and we had to run the script and change the code.

**Comment by Deepa-Puranik** [comment]
Yes, we did run DB update script. By updating EPiServer.CMS to version 12.11 fixed the issue but all the data getting erased for all the LinkItem field.
